scripts/lib/api/processHtml.ts

```diff
@@ -299,107 +297,155 @@ export function processMembersAndSetMeta(
           meta.apiName = id;
         }
 
-        if (apiType == "class") {
-          findByText($, $main, "em.property", "class").remove();
-          return `<span class="target" id="${id}"/><p><code>${$child.html()}</code>${github}</p>`;
-        }
+        return processMember($, $main, $child, $dl, priorApiType, apiType, id);
+      })
+      .join("\n");
 
-        if (apiType == "property") {
-          if (!priorApiType && id) {
-            $dl.siblings("h1").text(getLastPartFromFullIdentifier(id));
-          }
+    $dl.replaceWith(`<div>${replacement}</div>`);
+  }
+}
 
-          findByText($, $main, "em.property", "property").remove();
-          const signature = $child.find("em").text()?.replace(/^:\s+/, "");
-          if (signature.trim().length === 0) return;
-          return `<span class="target" id='${id}'/><p><code>${signature}</code>${github}</p>`;
-        }
+function processMember(
+  $: CheerioAPI,
+  $main: Cheerio<any>,
+  $child: Cheerio<any>,
+  $dl: Cheerio<any>,
+  priorApiType: string | undefined,
+  apiType: string,
+  id: string,
+) {
+  const githubSourceLink = prepareGitHubLink($child, apiType === "method");
 
-        if (apiType == "method") {
-          if (id) {
-            if (!priorApiType) {
-              $dl.siblings("h1").text(getLastPartFromFullIdentifier(id));
-            } else if (!$child.attr("id")) {
-              // Overload methods have more than one <dt> tag, but only the first one
-              // contains an id.
-              return `<p><code>${$child.html()}</code>${github}</p>`;
-            } else {
-              // Inline methods
-              $(`<h3>${getLastPartFromFullIdentifier(id)}</h3>`).insertBefore(
-                $dl,
-              );
-            }
-          }
-
-          findByText($, $main, "em.property", "method").remove();
-          return `<span class="target" id='${id}'/><p><code>${$child.html()}</code>${github}</p>`;
-        }
+  findByText($, $main, "em.property", apiType).remove();
```

Comment:
This call was common in all the `apiType` helper functions and I extracted it in the `processMember` function instead.

Reply:
I was going to ask if you're confident it's safe to do this because you're changing the sequence: often this call was at the end, not beginning. But looks like the API generation behaves the same, so I presume it's fine.

Reply:
Yes. At the beginning, I was wondering whether we were able to move it at the start of the function and it seems the script behaves the same. That line of code only removes the `apiType` from the beginning of each header, so it shouldn't be important for any other step